web/packages/core/src/ruffle-player.ts

```diff
@@ -1534,8 +1536,16 @@ export class RufflePlayer extends HTMLElement {
 
     protected debugPlayerInfo(): string {
         let result = `Allows script access: ${this.loadedConfig.allowScriptAccess}\n`;
+        let renderInfo = this._cachedDebugInfo;
         if (this.instance) {
-            result += `${this.instance.renderer_debug_info()}\n`;
+            try {
+                renderInfo = this.instance.renderer_debug_info();
+            } catch {
+                // ignored
```

Comment:
Could you make the output include something like "(Cached)", and maybe even the error message, when this happens?

Reply:
The error will be the whole "we're panicking and can't access stuff" that we currently spam the console with, I don't think it'll be valuable. I'll add Cached though!